src/internal/connector/exchange/restore_test.go

```diff
@@ -187,6 +187,18 @@ func (suite *ExchangeRestoreSuite) TestRestoreExchangeObject() {
 				return *folder.GetId()
 			},
 		},
+		{
+			name:     "Test Mail: Item Attachment_Mail",
```

Comment:
Can we add a test case for an item attachment that is a message that itself contains an attachment. To verify that our sanitization is not failing/panicing.